### PR TITLE
feature: Added Splat.Ninject

### PR DESCRIPTION
**Pull request by RLittlesII**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Adds a Ninject Splat Adapter

**What is the current behavior? (You can also link to an open issue here)**

There is no Ninject Adapter

**What is the new behavior (if this is a feature change)?**

There is a Ninject Adapter

**What might this PR break?**

Nothing

**Please check if the PR fulfills these requirements**
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: src/Splat.Ninject/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Splay.Ninject is an adapter for `IMutableDependencyResolver`.  It allows you to register your application dependencies in a Ninject `IKernel`.  You can then use the container as Splat's internal dependency resolver.
+
+### Register the Kernel
+
+```cs
+var kernel = new StandardKernel();
+kernel.Bind<IViewFor<MainViewModel>>().To<MainPage>();
+kernel.Bind<IViewFor<SecondaryViewModel>>().To<SecondaryPage>();
+kernel.Bind<MainViewModel>().ToSelf();
+kernel.Bind<SecondaryViewModel>()ToSelf();
```
Comment (worldbeater) [inline src/Splat.Ninject/README.md]: Missing `.` symbol

Comment (RLittlesII) [reply src/Splat.Ninject/README.md]: Fixed this.

File: src/NuGet.Config [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
```
Comment (glennawatson) [inline src/NuGet.Config]: We don't need a nuget.config file I don't think

Comment (RLittlesII) [reply src/NuGet.Config]: Not sure how this got in here.  It has been sneakily creating one without me knowing.  `git add .` is not my friend.

**Comment by RLittlesII** [comment]
@glennawatson The build is failing on the server, but not when I run it locally using `.\build.ps1` thoughts?

**Pull request merged**